### DISCUSSION TITLE
Cheat import not working

### DISCUSSION THREAD

**Discussion by Kingjuleyann911** [rafaelvcaetano/melonDS-android#1250] [Support]
Hello, I have tried to import cheats for Megaman Starforce Pegasus with the following xml file:

```<?xml version="1.0" encoding="UTF-8"?>
<codelist>
    <name>cheats nds</name>
    <game>
        <name>Mega Man Star Force PEGASUS</name>
      
 <gameid>A6AE 576C6233</gameid>
        <folder>
            <name>Cheats</name>
            <cheat>
                <name>Get Andromeda card + L.M.Shin brother</name>
                <note>Enter Network menu, go. to Wireless Client mode and choose Left Cell for L.M.Shin brother and Right Cell for Andromeda card</note>
                <codes>9218B8CC 00000100 0218B8D0 0000E030 0218B8CC D018420D D0000000 00000000</codes>
            </cheat>
        </folder>
    </game>
</codelist>
```

This file is located within a folder named "Cheats", inside the folder where the game's ROM is. When I try to use this file to import cheats it doesn't work. I have no idea about what is wrong except that the game id is good. What can I do to fix this issue ?

**Comment by rafaelvcaetano** [comment]
What exactly isn't working? Is some error generated when importing the cheat file? Or is the cheat file correctly imported but it doesn't appear?

**Reply by Kingjuleyann911** [reply]
It seems like the file is imported correctly (no errors or anything) but no cheats appear in the cheat menu

**Reply by rafaelvcaetano** [reply]
Thanks! I'll try to reproduce this.
